backend/editor/entries.py

```diff
@@ -270,20 +270,30 @@ async def list_projects(self, status=None):
             "MATCH (n:PROJECT)",
             "OPTIONAL MATCH (error_node:ERRORS {branch_name: n.branch_name, id: n.id})",
         ]
+
         params = {}
         if status is not None:
             # List only projects matching status
             query.append("WHERE n.status = $status")
             params["status"] = status
+
         query.extend(
             [
                 "WITH n, size(error_node.errors) AS errors_count",
                 "RETURN n{.*, errors_count: errors_count}",
                 "ORDER BY n.created_at",
             ]
         )
-        result = await get_current_transaction().run("\n".join(query), params)
-        return await async_list(result)
+
+        queryResult = await get_current_transaction().run("\n".join(query), params)
+
+        result = []
+
+        async for queryResultList in queryResult:
+            for item in queryResultList:
+                result.append(item)
```

Comment:
You can write it in a more pythonic way:

```suggestion
        result = [item async for result_list in query_result for item in result_list ]
```

Reply:
done, I was trying to avoid the double list comprehension hehe, but yeah, let's go with python style